text-ui-test/EXPECTED..txt

```diff
@@ -0,0 +1,170 @@
+Hello from
+ ____        _        
+|  _ \ _   _| | _____ 
+| | | | | | | |/ / _ \
+| |_| | |_| |   <  __/
+|____/ \__,_|_|\_\___|
```

Comment:
Perhaps you might want to generate custom ASCII art from a [website like this one](https://patorjk.com/software/taag/#p=display&h=0&v=0&f=Delta%20Corps%20Priest%201&t=Charizard).